src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java

```diff
@@ -221,6 +223,7 @@ public OperationCompletionRS updateTestItem(ReportPortalUser.ProjectDetails proj
 					INCORRECT_REQUEST,
 					"Unable to change status on test item with children"
 			);
+			checkInitialStatusAttribute(testItem);
```

Comment:
what about updating some tests according to the new logic?

Reply:
added